README.md

```diff
@@ -49,3 +49,12 @@ Here are some web pages that you may need to refer to:
       -https://www.kaggle.com/tathor/yugioh-trading-cards-dataset/data
       - YuGiOh is a card game that I have a lot of interest in and I think it would be fun to explore the most common card type or              monster attributes given the over 6,000 cards included in this data set.
   
+
+Cody Makinson
```

Comment:
Good. You have a bullet point for each dataset. Now organize each URL as a bullet point indented underneath one of those. The idea is that someone should be able to look at the information for each dataset separately.

Reply:
You should also put a dash in front of "Cody" and line it up under "- George Cowan" to make your name be at the right level of the outline.